LeaderboardBackend.Integration/HttpHelpers.cs

```diff
@@ -0,0 +1,26 @@
+using NUnit.Framework;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Text.Json;
+using System.Threading.Tasks;
+
+namespace LeaderboardBackend.Integration
+{
+	internal static class HttpHelpers
+	{
+		public static async Task<T> ReadFromResponseBody<T>(HttpResponseMessage response, JsonSerializerOptions jsonOptions) 
+		{ 
+			string rawJson = await response.Content.ReadAsStringAsync();
+			T? obj = JsonSerializer.Deserialize<T>(rawJson, jsonOptions);
+			Assert.NotNull(obj);
+			return obj!;
+		}
+
+		public static string ListToQueryString<T>(IEnumerable<T> list, string key)
```

Comment:
This method is not used in this PR, but I didn't want to go through the process of splitting up the file for the two PRs so it's going in here lol